editoast/src/modelsv2/rolling_stock_model/schedules_from_rolling_stock.rs

```diff
@@ -0,0 +1,70 @@
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+use crate::error::Result;
+use crate::modelsv2::rolling_stock_model::RollingStockModel;
+use crate::tables::{project, rolling_stock, scenario_v2, study, train_schedule_v2};
+use editoast_models::db_connection_pool::DbConnectionV2;
+
+#[derive(Debug, Serialize, Deserialize, ToSchema, PartialEq, Eq, PartialOrd, Ord)]
```

Comment:
The [`cfg_attr`](https://doc.rust-lang.org/reference/conditional-compilation.html#the-cfg_attr-attribute) attribute allows conditional compilation. Since these traits (`PartialEq, Eq, PartialOrd, Ord, Deserialize `) are only used in tests if (`cfg!(test)`), we can omit them when we're not testing.

Maybe I missed something but I don't think you'll have to change the tests to change these `derive(..)`s.